backend/corpora/parliament/utils/parlamint_v4.py

```diff
@@ -0,0 +1,322 @@
+from glob import glob
+from string import punctuation
+from typing import Iterable
+
+from addcorpus.python_corpora.extract import XML, Combined, Metadata
+from ianalyzer_readers.xml_tag import Tag
+from bs4.element import NavigableString, Tag as Node
+from bs4 import BeautifulSoup
+
+from addcorpus.es_mappings import non_indexed_text_mapping, keyword_mapping
+from addcorpus.python_corpora.corpus import FieldDefinition
+from addcorpus.python_corpora.filters import MultipleChoiceFilter
+from corpora.parliament.utils.parlamint import (
+    metadata_attribute_transform_func,
+    person_attribute_extractor,
+)
+
+
+"""
+This file was created as an updated utils file for the ParlaMint dataset, version 4.0. The previous utils file
+is based on version 2.0.
+"""
```

Comment:
Small note: module level docstrings should be the first line of the file. If you put it after the imports, it's not registered by `help`, code editors, etc.

I think by "the previous utils file", you're referring to `parliament.py`? This reference is ambiguous. Also, someone not familiar with the code history would not know which file came first chronologically.

If I were looking over a directory with `parliament.py` and `parliament_v4.py`, I would probably assume that `parliament.py` covers the most recent version and should be the default for new corpora, and the `v4` module is for compatibility with some older version.